firefox_puppeteer/api/places.py

```diff
@@ -4,6 +4,8 @@
 
 from collections import namedtuple
 
+from marionette_driver.errors import MarionetteException
```

Comment:
nit: do not add a full new line here, but add it to line 9 right before the other already imported `TimeoutException`.